tests/acceptance/customCommands/uploadRemote.js

```diff
@@ -0,0 +1,53 @@
+const path = require('path')
+const events = require('events')
+const archiver = require('archiver')
+
+class UploadRemote extends events.EventEmitter {
```

Comment:
```suggestion
class uploadRemote extends events.EventEmitter {
```

Reply:
In javascript, classes are usually PascalCase-d. And, you can use the custom command as `this.uploadRemote()` as usual due to the filename being `uploadRemote.js`.

Ref: https://www.robinwieruch.de/javascript-naming-conventions